operator/config/watcher/watcher.yaml

```
@@ -0,0 +1,23 @@
+# labels and selectors will be added using kustomize
+apiVersion: operator.kyma-project.io/v1alpha1
+kind: Watcher
+metadata:
+  name: watcher
```

this will conflict with other watcher crs created. you should call it
```suggestion
  name: controller-manager-watcher
  namespace: system
```
that way its creating these resources with the default values and will replace them together with the kustomization in `manager` and `default`